### ISSUE TITLE
Support Python 2 packages in workspace

### ISSUE THREAD

**Issue by murphm8** [colcon/colcon-bundle#61]
We currently only support parsing `setup.py` for Python 3. We should use other metadata from the `setup.py` to determine which Python version the package is for and install into the correct Python installation.

**Comment by murphm8** [comment]
As a temporary solution I have pushed a branch that allows for installation into Python 2 from a requirements.txt.

```
pip3 uninstall colcon-bundle
pip3 install git+https://github.com/colcon/colcon-bundle.git@pip_requirements
colcon bundle --pip-requirements requirements.txt
```

**Comment by babakc** [comment]
@abwiggins , assuming the master branch is what comes packaged with robomaker, this now works out of the box. Worked for me!

**Comment by babakc** [comment]
Yep, merged into master https://github.com/colcon/colcon-bundle/pull/62
